src/core/Akka.Tests/Actor/AskSpec.cs

```diff
@@ -141,6 +141,23 @@ public void Can_Ask_actor_inside_receive_loop()
             waitActor.Tell("ask");
             ExpectMsg("bar");
         }
+
+        [Fact]
+        public void Cancelled_ask_with_timeout_should_raise_timeout_exception()
```

Comment:
Can xunit supports async await. Can we please use it? This will help to avoid wrapping/unwrapping exceptions.